packages/stark-ui/src/modules/table/components/table.component.html

```diff
@@ -84,7 +82,7 @@
 			*ngFor="let col of columnProperties; trackBy: trackColumnFn"
 			[name]="col.name"
 			[headerLabel]="col.label"
-			[sortable]="col.isSortable"
+			[sortable]="col.isSortable !== false"
```

Comment:
I think the default value in `column.component.ts` is sufficient to set the default to true, also we might show this on the demo page by removing `isSortable: true` from the columnProperties

Reply:
This issue is: 

if col.isSortable = undefined, column receives sortable=false. 
So, the default value `true` is not applied 😕 

That's why I added this check...